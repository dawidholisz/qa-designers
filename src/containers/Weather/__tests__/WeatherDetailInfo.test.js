import React from 'react'
import { render } from '@testing-library/react'
import WeatherDetailsInfo from '../WeatherDetailInfo'

const renderComponent = props => {
  return render(<WeatherDetailsInfo {...props} />)
}

describe('<WeatherDetailsInfo/>', () => {
  it('render detailsInfo with number value', () => {
    const { container } = renderComponent({ value: 15, label: 'Label' })
    expect(container).toMatchSnapshot()
  })
  it('render detailsInfo with string value', () => {
    const { container } = renderComponent({ value: 'Value', label: 'Label' })
    expect(container).toMatchSnapshot()
  })
  it('render detailsInfo with unit', () => {
    const { container } = renderComponent({ value: 'Value', label: 'Label', unit: 'Unit' })
    expect(container).toMatchSnapshot()
  })
})
