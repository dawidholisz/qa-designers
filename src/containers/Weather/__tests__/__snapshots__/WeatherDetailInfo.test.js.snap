// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherDetailsInfo/> render detailsInfo with number value 1`] = `
<div>
  <div
    class="weather-details-info"
  >
    <p
      class="weather-details-info__value"
    >
      15
    </p>
    <p
      class="weather-details-info__label"
    >
      Label
    </p>
  </div>
</div>
`;

exports[`<WeatherDetailsInfo/> render detailsInfo with string value 1`] = `
<div>
  <div
    class="weather-details-info"
  >
    <p
      class="weather-details-info__value"
    >
      Value
    </p>
    <p
      class="weather-details-info__label"
    >
      Label
    </p>
  </div>
</div>
`;

exports[`<WeatherDetailsInfo/> render detailsInfo with unit 1`] = `
<div>
  <div
    class="weather-details-info"
  >
    <p
      class="weather-details-info__value"
    >
      Value
      Unit
    </p>
    <p
      class="weather-details-info__label"
    >
      Label
    </p>
  </div>
</div>
`;
